src/main/java/no/template/Application.java

```diff
@@ -14,6 +14,7 @@
 @SpringBootApplication
 public class Application {
 
+
```

Comment:
Var dette veldig nødvendig?

Reply:
Nei, det var ikke nødvendig. Det var jeg som la inn en liten endring for å trigge et bygg, og det ble liggende igjen et linjeskift. Jeg fjerner det før jeg merger.